emukit/bayesian_optimization/acquisitions/expected_improvement_per_cost.py

```diff
@@ -0,0 +1,61 @@
+from typing import Tuple, Union
+
+import numpy as np
+
+from ...core.interfaces import IModel, IDifferentiable
+from ...core.acquisition import Acquisition
+from emukit.bayesian_optimization.acquisitions import ExpectedImprovement
+
+
+class ExpectedImprovementPerCost(Acquisition):
+
+    def __init__(self, model: Union[IModel, IDifferentiable], cost_model: Union[IModel, IDifferentiable],
+                 jitter: np.float64 = np.float64(0))-> None:
+        """
+        This acquisition computes for a given input the improvement over the current best observed value in
+        expectation. For more information see:
+
+        Efficient Global Optimization of Expensive Black-Box Functions
+        Jones, Donald R. and Schonlau, Matthias and Welch, William J.
+        Journal of Global Optimization
+
+        :param model: model that is used to compute the improvement.
+        :param jitter: parameter to encourage extra exploration.
+        """
+
+        self.model = model
+        self.cost_model = cost_model
+        self.jitter = jitter
+
+        self.ei = ExpectedImprovement(model, jitter)
+
+    def evaluate(self, x: np.ndarray) -> np.ndarray:
+        """
+        Computes the Expected Improvement.
+
+        :param x: points where the acquisition is evaluated.
+        """
+        improvement = self.ei.evaluate(x)
+
+        mean, _ = self.cost_model.predict(x)
```

Comment:
let's give this thing another name. maybe cost instead if mean?

Reply:
agreed